CONTRIBUTING.md

```diff
@@ -13,3 +13,41 @@ the GitHub repository. Generally, we expect two maintainers to review your pull
 request before it is approved for merging. For more details, see the
 [MAINTAINERS](MAINTAINERS.md) page.
 
+**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
```

Comment:
I am not sure if we are supposed to edit this file or not...

Reply:
let's leave it alone